pkgs/applications/networking/blocky/default.nix

```
@@ -21,6 +21,8 @@ buildGoModule rec {
 
   vendorSha256 = "sha256-fsMBL9qyhIrV6eAsqpSaNniibMdYRVBnl2KCzStvMGQ=";
 
+  ldflags = [ "-X github.com/0xERR0R/blocky/util.Version=${version}" ];
```

```suggestion
  ldflags = [ "-s" "-w" "-X github.com/0xERR0R/blocky/util.Version=${version}" ];
```
lets also tell go to strip

---

@SuperSandro2000: Thanks, I didn't know. (I couldn't find `-w` option in `man ld`, but apparently it disables DWARF debug info.) But I feel that's off topic for "embed version info". Longer term, maybe `ldflags = ["-s" "-w"]` is something buildGoModule should default to?

---

https://github.com/NixOS/nixpkgs/issues/177698